build/manifest/main.go

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) 2023-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Why it's 2023? normally it's been 2015.

Reply:
Thanks for asking, @isacikgoz! Legal has asked us to anchor the year to the repository, as it's otherwise less effective a copyright claim when none of the code backing this "program" existed back in 2015. Fortunately, we /don't/ need to keep this accurate to a file-level.